### ISSUE TITLE
Schema Subset Generation Tool (SSGT) Does not Load

### ISSUE THREAD

**Issue by brettforbes** [NIEM/NIEM.github.io#162]
When trying to access your Schema Subset Generation Tool (SSGT), it will not load. I am using a basic Microsoft Surface laptop on a home network. Every other website seems to load correctly, only this one is a problem.

When i click on the link from this [page](https://webb.github.io/NIEM.github.io/reference/tools/ssgt/), I receive the response shown in the image below. The response  is the same regardless of browser  chosen. Why?

![image](https://github.com/NIEM/NIEM.github.io/assets/24556510/91fe3a3e-7e3c-4f1b-8cb7-d68db64f50fa)

Basically, the page does not work. I get the same response on both Edge and Chrome. Why?

**Comment by cdmgtri** [comment]
Hi.  We just updated some settings and think it should be working for you now.  Could you please try it again?

**Comment by brettforbes** [comment]
It worked. Thanks a lot. 

My browser is asking me to allow cookies, how do i do this just for your website? I can use either Edge or Chrome

**Comment by cdmgtri** [comment]
If you block all cookies by default, you can enable them for specific sites.

Chrome: https://support.google.com/chrome/answer/14114868 

Edge: https://support.microsoft.com/en-us/microsoft-edge/view-cookies-in-microsoft-edge-a7d95376-f2cd-8e4a-25dc-1de753474879
